packages/config/src/options.js

```diff
@@ -320,7 +321,12 @@ export function getNuxtConfig(_options) {
 
   // include SFCs in node_modules
   options.build.transpile = [].concat(options.build.transpile || [])
-    .map(module => module instanceof RegExp ? module : new RegExp(module))
+    .map(module => (module instanceof RegExp)
+      ? module
+      : new RegExp(
+        escapeRegExp(path.normalize(module))
```

Comment:
`normalize` won't parse windows style path into POSIX, so it may still not work in windows.

Reply:
@clarkdo,

I think you meant to say "it may still not work in POSIX"?

Here is my proposal for this issue: e85a201
>  Convert to POSIX path before normalisation
>  
>  path.normalize can convert POSIX to Windows but not Windows to POSIX.
>  This replacement is added to ensure that the path is always in POSIX
>  form before normalisation occur.

Reply:
Please see above comment 😄